src/Cvl/LDAPWrapper/LDAPWrapper.php

```diff
@@ -9,6 +9,7 @@ class LDAPWrapper {
 	const LDAP_ATTRIBUTE_MANAGER = 'groupManager';
 	const LDAP_ATTRIBUTE_DESCRIPTION = 'description';
 	const LDAP_ATTRIBUTE_ACCOUNT_NAME = 'sAMAccountName';
+	const LDAP_MATCHING_RULE_BIT_AND = '1.2.840.113556.1.4.803';
```

Comment:
I've lost 30 minutes exploring what that string means and it's BIT AND rule...
Check it [here](http://www.ldapexplorer.com/en/manual/109010000-ldap-filter-syntax.htm) :)